.github/ISSUE_TEMPLATE/module_proposal_bicep_child.yml

```
@@ -0,0 +1,228 @@
+name: New Bicep Child Module Proposal 🗒️
+description: Want a new child module of a Bicep resource module to exist? Let us know!
```

As discussed offline, I'd suggest to clarify that this issue is to propose publishing a child module rather than developing a new one. For example:
```suggestion
description: Want to publish the child module of a Bicep resource module? Let us know!
```

---

Suggestion applied.